Assets/Tests/EditMode/LightSwitchTests.cs

```diff
@@ -9,15 +9,17 @@
 public class LightsSwitchTests
 {
     [Test]
-    public void IncrementValueOnMoveNext()
+    public void InfiniteEnumerator_IncrementValueOnMoveNext()
     {
-        int initialValue = 3;
-        var enumerator = new InfiniteEnumerator(initialValue);
-
+        var enumerator = new InfiniteEnumerator(3);
         enumerator.MoveNext();
-        int currentValue = enumerator.Current;
+        Assert.AreEqual(3, enumerator.Current);
```

Comment:
Oh yes good catch!

Reply:
Also I just realised I declared enumerator twice haha and I meant to use the first declaration instead of the duplicate which is the reason why it's there!